Assets/Scripts/SS3D/Systems/Chat/ToggleChats.cs

```diff
@@ -0,0 +1,27 @@
+﻿using UnityEngine;
+
+namespace SS3D.Engine.Chat
+{
+    public class ToggleChats : MonoBehaviour
+    {
+        [SerializeField]
+        private Transform chatWindowsContainer;
+        
+        public void ToggleAllChats()
+        {
+            InGameChatWindow[] allChatWindows = chatWindowsContainer.GetComponentsInChildren<InGameChatWindow>();
```

Comment:
Can't you use View to get all the chat windows instead of using an expensive GetComponentsInChildren call?